Packages/com.llealloo.audiolink/Runtime/Scripts/AudioReactiveLight.cs

```diff
@@ -1,19 +1,23 @@
 ﻿using UnityEngine;
 
+#if UDONSHARP
+using UdonSharp;
+#endif
+
 namespace AudioLink
 {
+    [RequireComponent(typeof(Light))]
 #if UDONSHARP
-    using UdonSharp;
-
+    [UdonBehaviourSyncMode(BehaviourSyncMode.NoVariableSync)]
```

Comment:
Is this intentional?

Reply:
Yes. The behaviour has no sync data on it, so there is no need to have the inspector display the sync data mode options. NoVariableSync allows other scripts on the object to use sync data without interferring, unlike `None` mode.